providers/teams/pages/members/[id].tsx

```diff
@@ -0,0 +1,215 @@
+import React, { ChangeEvent, useEffect, useState } from 'react'
+import { useRouter } from "next/router"
+import dynamic from "next/dynamic"
+import { gql } from "apollo-boost"
+import { apolloClient } from '../../lib/apollo'
+
+
+const getTeamMembers = gql`
+query getTeamMembers($teamId: Int!) {
+    team_members(where: {teamId: {_eq: $teamId}}) {
```

Comment:
Do we have to have `_eq`? isn't it possible to just `{ teamId: $teamId }`?

Reply:
I don't think we can, double check me [here](https://hasura.io/docs/1.0/graphql/manual/queries/query-filters.html#equality-operators-eq-neq)

Reply:
fuck it